src/index.js

```diff
@@ -0,0 +1,96 @@
+// change the scene depending on the sky
+// make a reset city name button
+// add a "landscape" element
+
+const state = {
+  clickCount: 50,
+};
+
+const colors = (clicks, text) => {
+  if (clicks < 50) {
+    text.style.color = 'blue';
+  } else if (clicks < 59) {
+    text.style.color = 'green';
+  } else if (clicks < 69) {
+    text.style.color = 'pink';
+  } else if (clicks < 79) {
+    text.style.color = 'orange';
+  } else {
+    text.style.color = 'red';
+  }
+};
+
+const landscape = (clicks, container) => {
+  if (clicks < 50) {
+    container.textContent = '🌲🌲⛄️🌲⛄️🍂🌲🍁🌲🌲⛄️🍂🌲';
+  } else if (clicks < 59) {
+    container.textContent = '🍁🍃🍂🍁🍃🍂🍁🍃🍂🍁🍃🦙';
+  } else if (clicks < 69) {
+    container.textContent = '🌾🌾_🍃_🪨__🛤_🌾🌾🌾_🍃';
+  } else if (clicks < 79) {
+    container.textContent = '🌸🌿🌼__🌷🌻🌿_☘️🌱_🌻🌷';
+  } else {
+    container.textContent = '🌵__🐍_🦂_🌵🌵__🐍_🏜_🦂';
+  }
+};
+const increaseClickCount = () => {
+  const displayTemp = document.getElementById('temp');
+  const landscapeDisplay = document.getElementById('landscape-scene');
+  state.clickCount += 1;
+  displayTemp.textContent = state.clickCount;
+  console.log('up');
+  colors(state.clickCount, displayTemp);
+  landscape(state.clickCount, landscapeDisplay);
+};
+
+const decreaseClickCount = () => {
+  const displayTemp = document.getElementById('temp');
+  const landscapeDisplay = document.getElementById('landscape-scene');
+  state.clickCount -= 1;
+  displayTemp.textContent = state.clickCount;
+  console.log('down');
+  colors(state.clickCount, displayTemp);
+  landscape(state.clickCount, landscapeDisplay);
+};
+
+const displayCity = () => {
+  console.log('test');
+  document.getElementById('city-title').textContent =
+    document.getElementById('city-name').value;
+  return false;
+};
+
+const removeCity = () => {
+  console.log('test');
+  document.getElementById('city-title').textContent = 'City';
+  return false;
+};
+
+const skyChange = () => {
+  let skyDisplay = document.getElementById('sky-scene');
+  let currentSky = document.getElementById('sky').value;
+  if (currentSky === 'sunny') {
+    skyDisplay.textContent = '☁️ ☁️ ☁️ ☀️ ☁️ ☁️☁️ ☁️ ☁️ ☀️ ☁️';
+  } else if (currentSky === 'rainy') {
+    skyDisplay.textContent = '🌧🌈⛈🌧🌧💧⛈🌧🌦🌧💧🌧';
+  } else if (currentSky === 'snowy') {
+    skyDisplay.textContent = '🌨❄️🌨🌨❄️❄️🌨❄️🌨❄️❄️🌨🌨';
+  } else if (currentSky === 'cloudy') {
+    skyDisplay.textContent = '☁️☁️ ☁️ ☁️☁️ ☁️ 🌤 ☁️ ☁️☁️';
+  }
+};
```

Comment:
Consider the above logic and how similar all the branches are. We look for whether the selected sky matches a certain value, and then we set the text to display for the sky. What if we had a list of records that we could iterate through to find the new sky value. We could set up something like
```js
  const skyRecords = [
    { skyValue: "sunny", display: "☁️ ☁️ ☁️ ☀️ ☁️ ☁️☁️ ☁️ ☁️ ☀️ ☁️"},
    { skyValue: "some other sky value", display: "some other display"},
    ...
  ];
```
if we wanted to iterate through, or something like
```js
  const skyRecords = {
     "sunny": "☁️ ☁️ ☁️ ☀️ ☁️ ☁️☁️ ☁️ ☁️ ☀️ ☁️",
     "some other sky value": "some other display",
    ...
  };
```
for direct lookups (this wouldn't work if we tried to apply this to the equivalent updates for temperature ranges, which would work better with an iterative comparison).

All we need to do is look up the new display string and set it. Writing our code to be more data-driven would potentially allow us to populate these lists from a web service, allowing for new values without needing to modify the code!